sdk/datafactory/Azure.ResourceManager.DataFactory/src/autorest.md

```diff
@@ -87,4 +87,101 @@ directive:
       $['/subscriptions/{subscriptionId}/providers/Microsoft.DataFactory/locations/{locationId}/getFeatureValue'].post.parameters[3].name = 'content';
       $['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/getFeatureValue'].post.parameters[4].name = 'content';
 
+  # keeping the payload consistent with RestApi (32 derived classes of LinkedService)
+  - from: LinkedService.json
+    where: $.definitions.AzureStorageLinkedServiceTypeProperties.properties.connectionString
+    transform: $.type = "string"
+  - from: LinkedService.json
```

Comment:
I don't think we can change the type here because the service team has some special usage, can you change it back?

Reply:
BinaryData support pure string, not a json string, currently there has a bug on Serialization, but can workaround by add `\"`

Reply:
fixed